src/components/pages/Offer/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -545,35 +595,41 @@ const OfferForm = ({
               {'Accessibilité'}
             </h3>
             <p className="section-description">
-              {'Cette offre est accessible aux publics en situation de :'}
+              {'Cette offre est accessible aux publics en situation de handicapes :'}
             </p>
             <CheckboxInput
               SvgElement={VisualDisabilitySvg}
               checked={formValues.visualDisabilityCompliant || false}
               label="Handicap visuel"
               name="visualDisabilityCompliant"
-              onChange={handleSingleFormUpdate}
+              onChange={handleDisabilityCompliantUpdate}
             />
             <CheckboxInput
               SvgElement={MentalDisabilitySvg}
               checked={formValues.mentalDisabilityCompliant || false}
               label="Handicap mental"
               name="mentalDisabilityCompliant"
-              onChange={handleSingleFormUpdate}
+              onChange={handleDisabilityCompliantUpdate}
             />
             <CheckboxInput
               SvgElement={MotorDisabilitySvg}
               checked={formValues.motorDisabilityCompliant || false}
               label="Handicap moteur"
               name="motorDisabilityCompliant"
-              onChange={handleSingleFormUpdate}
+              onChange={handleDisabilityCompliantUpdate}
             />
             <CheckboxInput
               SvgElement={AudioDisabilitySvg}
               checked={formValues.audioDisabilityCompliant || false}
               label="Handicap auditif"
               name="audioDisabilityCompliant"
-              onChange={handleSingleFormUpdate}
+              onChange={handleDisabilityCompliantUpdate}
+            />
+            <CheckboxInput
+              checked={formValues.noDisabilityCompliant || false}
```

Comment:
```suggestion
              checked={formValues.noDisabilityCompliant}
```

Pareil pour les précédentes